de-lib/src/main/java/org/iplantc/de/theme/base/client/desktop/DesktopMessages.properties

```diff
@@ -41,3 +41,24 @@ sectionOne = Section 1
 resetHpc = Reset HPC Token
 resetHpcPrompt = <p>Click `Reset HPC Token` to reset your HPC token:</p>
 hpcResetFailure = Unable to reset your HPC token. Please contact support!
+webhooks=Webhooks
+webhooksPrompt = Add incoming webhook integration for slack. DE will post notifications to this URL:
+dataNotification=Data (e.g: When someone shares a file with you. )
+appsNotification=Apps (e.g: When someone shares an app with you. )
+analysesNotification=Analyses (e.g: When your analysis status changes.)
+toolsNotification=Tools (e.g: When your tool request is completed.)
+permNotification=Permanent Identifiers (e.g: When your permanent identifier request is completed.)
+teamNotification=Teams (e.g: When you are added to a team.)
+hookTopic=Select topics:
+test=Test
+webhookSaveError = Unable to save webhooks. Please try again!
+testWebhookFail = Unable to test webhook. Please check the webhook url again and try to add again!
+testWebhookSuccess = Please check your slack to see if you have received a test message!
+mustSelectATopic=You must select at least one topic for webhook notification! 
```

Comment:
The `!` in these messages seem a little extreme ❗️ 
I'd use just a period in these messages.